.mergify.yml

```
@@ -194,8 +194,27 @@ pull_request_rules:
     conditions:
       - or:
           - and:
-              - label!=DNM
               - base~=^(release-.+)$
```

```suggestion
              - label!=DNM
              - base~=^(release-.+)$
              - "#approved-reviews-by>=2"
              - "#changes-requested-reviews-by=0"
              - "approved-reviews-by=@ceph/ceph-csi-contributors"
              - "approved-reviews-by=@ceph/ceph-csi-maintainers"
```

@nixpanic 
We need these rules too ?

---

It's just a copy/paste from the whole rule-set that was there. The diff looks a little awkward, but I think we need them all?

---

ah, you added something. Yes, we need those too! I'll update the PR in a bit.